README.md

```diff
@@ -51,4 +55,87 @@ Fetches raw data and returns JSON,
 ```
 SSSDK.shared.fetchData(by query:)
 ```
+## Installation
 
+#### Swift Package Manager
+
+SSSDK is available through [Swift Package Manager](https://swift.org/package-manager/).
+
++ For App integration
+
+#### Swift Package Manager
+You can use [Swift Package Manager](https://swift.org/package-manager/) to install SwiftKeychainWrapper using Xcode:
+
+1. Open your project in Xcode
+
+2. Click "File" -> "Swift Packages" -> "Add Package Dependency..."
+
+3. Paste the following URL: https://github.com/deveronuas/sssdk.git
+
+4. Click "Next" -> "Next" -> "Finish"
+
+## General Usage
+import SwiftUI
+import SimpleSalesforceSDK
+
+struct OAuthView: View {
+  let host = "your host url"
+  let clientId = "your client's id"
+  let clientSecret = "your client's secret"
+  let redirectUri = "your redirect uri"
+  let fetchQuery = "your SOQL query to fetch the data"
+
+  @State private var launchLoginView = false
+  init(){
+    // Required -> To use the SSSDK framework you must configure it.
+    SSSDK.shared.configure(host: host, redirectUri: redirectUri, clientId: clientId, clientSecret: clientSecret)
+  }
+  var body: some View {
+    VStack{
+      Button{
+        launchLoginView.toggle()
+      }label: {
+        Text("Login")
+      }
+      .sheet(isPresented: $launchLoginView) {
+        try? SSSDK.shared.login()
+          .onOpenURL(perform: { url in
```

Comment:
not sure if this should be here... looks weird that this `onOpenURL` is attached to the login view

Reply:
I tried that we can use it anywhere but I can move it outside of that modal sheet.